Change the group of users who can access alsa devices from 'audio' to 'sound'
since we already use 'sound' for other sound devices.

--- alsa-lib-1.0.10/src/conf/pcm/dsnoop.conf~	2005-09-08 08:36:00.000000000 +0200
+++ alsa-lib-1.0.10/src/conf/pcm/dsnoop.conf	2006-03-14 21:35:56.000000000 +0100
@@ -26,7 +26,7 @@
 	}
 	type dsnoop
 	ipc_key 5778293
-	ipc_gid audio
+	ipc_gid sound
 	ipc_perm 0660
 	slave {
 		pcm {
--- alsa-lib-1.0.10/src/conf/pcm/dmix.conf~	2005-09-19 14:38:14.000000000 +0200
+++ alsa-lib-1.0.10/src/conf/pcm/dmix.conf	2006-03-14 21:36:03.000000000 +0100
@@ -26,7 +26,7 @@
 	}
 	type dmix
 	ipc_key 5678293
-	ipc_gid audio
+	ipc_gid sound
 	ipc_perm 0660
 	slave {
 		pcm {
